Update quick start (#232)
AY2122S1-CS2103T-W16-2/tp: docs/UserGuide.md

{:.caption}

#### 2.2.2 Inventory & Statistics Tab

Displays [inventory](#32-inventory) and [statistics](#35-statistics).

![Inventory & Statistics tab](images/product-screenshots/general/BarChart.png)

Fig 2. Inventory & Statistics tab
{:.caption}

### 2.3 Creating your first order

This is a step-by-step guide to creating your first order. If you have not installed BobTheBistroBoss, you
may do so by following the instructions [here](#21-installation).

For this guide, we will simulate the following scenario:

Your regular customer John Doe wishes to place two orders, one on 10 December 2021 and the other on 11 December 2021.
Both of these orders are to be delivered at 12pm. For both of these orders, he will require 3 tuna sandwiches. Each
tuna sandwich will be priced at $2.

Here are the details of John Doe:
* Name: John Doe
* Phone Number: 92345678
* Email: johndoe@gmail.com
* Address: Blk 422, Bedok North Road, #15-25

Here are the ingredients that will be used for the tuna sandwich:

* 2 slices of bread
* 1 slice of cheese
* 1 can of tuna

> Suppose you currently have these ingredients with you but have not been added in the inventory:

Reviewer: ```suggestion
Suppose you currently have these ingredients with you:
```

We can go back to the previous one and make this section more like a story

* 100 slices of bread
* 100 slices of cheese
* 100 cans of tuna

#### 2.3.1 Approach 1

We can proceed to add the first order on 10 December 2021 by entering the following
command in the command box at the bottom of the app:

`add-o cn/John Doe cp/92345678 ca/Blk 422, Bedok North Road, #15-25 rn/Tuna Sandwich ri/Bread-2-slices, Cheese-1-slices, Tuna-1-cans op/6 oq/3 od/10-12-2021 1200`

You will notice that the order has been added to the Orders list (orders are sorted by deadline). However, this is not
the recommended approach, especially for repeated orders, as it can be very inefficient. Moreover, this approach does
not allow you to track ingredients in your inventory. We will now add the second order using our recommended approach.

#### 2.3.2 Approach 2 (Recommended)

Firstly, switch over to the Inventory & Statistics tab by entering the following command:

`tab 2`

> Now, we will proceed to add our 3 ingredients to the inventory by entering the following commands:

Reviewer: ```suggestion
Remember the ingredients you had? We will proceed to add them to the inventory by entering the following commands:
```

`add-i in/Bread iq/100 iu/slices`

`add-i in/Cheese iq/100 iu/slices`

`add-i in/Tuna iq/100 iu/cans`

> 

Reviewer: All these commands should be updated because after the first order you would have less quantity

You will notice that the ingredients, bread, cheese and tuna have been added to the Inventory list
(ingredients are sorted alphabetically).

Next, we will switch back to the Home tab by entering the following command:

`tab 1`

Let's start by adding our recipe to the Recipe Bookmarks list by entering the following command:

`add-r rn/Tuna Sandwich ri/Bread-2-slices, Cheese-1-slices, Tuna-1-cans rp/2`

You will notice that the recipe has been added to the Recipe Bookmarks list (recipes are sorted alphabetically). Take
note of the index of the recipe bookmark.

Now, we will also add our client to the Client Bookmarks list by entering the following command:

`add-c cn/John Doe cp/92345678 ce/johndoe@gmail.com ca/Blk 422, Bedok North Road, #15-25`

You will notice that the client has been to the Client Bookmarks list (clients are sorted alphabetically). Take

> note of the index of the client bookmark.

Reviewer: ```suggestion
note of the index of the client bookmark. We will refer to this index as `CLIENT_INDEX`.
```

Finally, we will add our order by entering the following command:

> `add-o c/CLIENT_BOOKMARK_INDEX r/RECIPE_BOOKMARK_INDEX oq/3 od/11-10-2021 1200`

Reviewer: ```suggestion
`add-o c/CLIENT_INDEX r/RECIPE_INDEX oq/3 od/11-10-2021 1200`
```

> Replace `CLIENT_BOOKMARK_INDEX` and `RECIPE_BOOKMARK_INDEX` with the respective indexes you took note of in the

Reviewer: ```suggestion
Replace `CLIENT_INDEX` and `RECIPE_INDEX` with the respective indexes you took note of in the
```

{:.caption}

#### 2.2.2 Inventory & Statistics Tab

Displays [inventory](#32-inventory) and [statistics](#35-statistics).

![Inventory & Statistics tab](images/product-screenshots/general/BarChart.png)

Fig 2. Inventory & Statistics tab
{:.caption}

### 2.3 Creating your first order

This is a step-by-step guide to creating your first order. If you have not installed BobTheBistroBoss, you
may do so by following the instructions [here](#21-installation).

For this guide, we will simulate the following scenario:

> John Doe wishes to place two orders, one on 10 October 2021 and the other on 11 October 2021. Both of these orders are

Reviewer: Can you make this more user-centric. Something like your regular customer John Doe....

to be delivered at 12pm. For both of these orders, he will require 3 tuna sandwiches. Each tuna sandwich will be priced
at $2.

Here are the details of John Doe:
* Name: John Doe
* Phone Number: 92345678
* Email: johndoe@gmail.com
* Address: Blk 422, Bedok North Road, #15-25

Here are the ingredients that will be used for the tuna sandwich:

* 2 slices of bread
* 1 slice of cheese
* 1 can of tuna

> Here are the quantities of the respective ingredients in our inventory:

Reviewer: ```suggestion
Suppose you currently have these items in your inventory:
```

* 100 slices of bread
* 100 slices of cheese
* 100 cans of tuna

#### 2.3.1 Approach 1

We can proceed to add the first order on 10 October 2021 by entering the following
command in the command box at the bottom of the app:

> `add-o cn/John Doe cp/92345678 ca/Blk 422, Bedok North Road, #15-25 rn/Tuna Sandwich ri/Bread-2-slices,Cheese-1-slices,Tuna-1-cans op/6 oq/3 od/10-10-2021 1200`

Reviewer: ```suggestion
`add-o cn/John Doe cp/92345678 ca/Blk 422, Bedok North Road, #15-25 rn/Tuna Sandwich ri/Bread-2-slices, Cheese-1-slices, Tuna-1-cans op/6 oq/3 od/10-10-2021 1200`
```

> You will notice that the order has been added at the 4th index in the Orders list. However, this is not the

Reviewer: These indexes are hardcoded. I think it is best to remove this so that we dont have to worry about changing this in the future if we were to remove some of the sample data stuff

recommended approach, especially for repeated orders, as it can be very inefficient. Moreover, this approach does not
allow you to track ingredients in your inventory. We will now add the second order using our recommended approach.

#### 2.3.2 Approach 2 (Recommended)

Firstly, switch over to the Inventory & Statistics tab by entering the following command:

`tab 2`

Now, we will proceed to add our 3 ingredients to the inventory by entering the following commands:

`add-i in/Bread iq/100 iu/slices`

`add-i in/Cheese iq/100 iu/slices`

`add-i in/Tuna iq/100 iu/cans`

> You will notice that the ingredients, bread, cheese and tuna has been added at the 3rd, 4th, and 11th indexes

Reviewer: Same problem with the indexes

in the Inventory list respectively.

Next, we will switch back to the Home tab by entering the following command:

`tab 1`

Let's start by adding our recipe to the Recipe Bookmarks list by entering the following command:

> `add-r rn/Tuna Sandwich ri/Bread-2-slices,Cheese-1-slices,Tuna-1-cans rp/2`

Reviewer: ```suggestion
`add-r rn/Tuna Sandwich ri/Bread-2-slices, Cheese-1-slices, Tuna-1-cans rp/2`
```

> You will notice that the recipe has been added at the 9th index in the Recipe Bookmarks list.

Reviewer: Same here.

Now, we will also add our client to the Client Bookmarks list by entering the following command:

`add-c cn/John Doe cp/92345678 ce/johndoe@gmail.com ca/Blk 422, Bedok North Road, #15-25`

> You will notice that the client has been added at the 6th index in the Client Bookmarks list.

Reviewer: Same here. Dont need to say the index.

Finally, we will add our order by entering the following command:

> `add-o c/6 rn/Tuna Sandwich r/9 oq/3 od/11-10-2021 1200`

Reviewer: Why's the recipe name stated here again?

> You will notice that the order has been added at the 5th index in the Orders list. This approach is more efficient as

Reviewer: Same problem with the index.
